diff --git a/CMakeLists.txt b/CMakeLists.txt
index f2250bc..3396a19 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -12,13 +12,18 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
 endif()
 
 find_package(ament_cmake REQUIRED)
+find_package(builtin_interfaces REQUIRED)
 find_package(rosidl_default_generators REQUIRED)
 
 set(msg_files
   "msg/UUID.msg"
 )
 
-rosidl_generate_interfaces(${PROJECT_NAME} ${msg_files})
+rosidl_generate_interfaces(${PROJECT_NAME}
+  ${msg_files}
+  DEPENDENCIES builtin_interfaces
+  ADD_LINTER_TESTS
+)
 
 ament_export_dependencies(rosidl_default_runtime)
 
diff --git a/package.xml b/package.xml
index 1813222..150b02f 100644
--- a/package.xml
+++ b/package.xml
@@ -16,6 +16,9 @@
 
   <buildtool_depend>rosidl_default_generators</buildtool_depend>
 
+  <build_depend>builtin_interfaces</build_depend>
+
+  <exec_depend>builtin_interfaces</exec_depend>
   <exec_depend>rosidl_default_runtime</exec_depend>
 
   <test_depend>ament_lint_auto</test_depend>
