diff --git a/src/qt_gui_cpp_sip/CMakeLists.txt b/src/qt_gui_cpp_sip/CMakeLists.txt
index dc27bd8..a6677c8 100644
--- a/src/qt_gui_cpp_sip/CMakeLists.txt
+++ b/src/qt_gui_cpp_sip/CMakeLists.txt
@@ -66,6 +66,7 @@ if(sip_helper_FOUND)
   set(qt_gui_cpp_BINDINGS "${qt_gui_cpp_BINDINGS}" PARENT_SCOPE)
 
   build_sip_binding(qt_gui_cpp_sip qt_gui_cpp.sip
+    SIP_CONFIGURE ${CMAKE_CURRENT_SOURCE_DIR}/sip_configure.py
     SOURCE_DIR ${PROJECT_SOURCE_DIR}/src/qt_gui_cpp_sip
     LIBRARY_DIR ${CMAKE_CURRENT_BINARY_DIR}
     DEPENDS ${qt_gui_cpp_sip_DEPENDENT_FILES}
